new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java

```diff
@@ -55,4 +62,54 @@ public static boolean doesFileExist(String fileName) {
     }
     return false;
   }
+  
+  /**
+   * Remove the given directory and its contents.
+   *
+   * @param dir the directory to be cleaned up
+   */
+  public static void cleanupDirectory(String dir) throws IOException {
+    File file = new File(dir);
+    logger.info("Cleaning up directory " + dir);
+    if (!file.exists()) {
+      // nothing to do
+      return;
+    }
+
+    if (!file.isDirectory()) {
+      throw new IllegalArgumentException("The parameter " + dir + " should be a directory.");
+    }
+
+    Files.walk(Paths.get(dir))
```

Comment:
we have commons-io in the pom, so you could do this with just one call to `FileUtils.deleteDirectory(file)`

Reply:
That is good to know. I'll change it.